components/src/atoms/Toast/index.tsx

```diff
@@ -367,59 +366,78 @@ export function Toast(props: ToastProps): JSX.Element {
               whiteSpace={NO_WRAP}
             >
               {headingText}
-            </LegacyStyledText>
+            </StyledText>
           ) : null}
           <Flex alignItems={ALIGN_CENTER}>
-            <LegacyStyledText css={TEXT_STYLE}>{message}</LegacyStyledText>
-            {linkText != null ? (
-              <Link
-                role="button"
-                onClick={() => {
-                  onLinkClick()
-                  onCloseHandler()
-                }}
-                css={TEXT_STYLE}
-                marginLeft={SPACING.spacing4}
-                marginRight={SPACING.spacing8}
-                textDecoration={TYPOGRAPHY.textDecorationUnderline}
-              >
-                {linkText}
-              </Link>
-            ) : null}
+            <StyledText css={TEXT_STYLE}>{message}</StyledText>
```

Comment:
according to figma, looks like even TEXT_STYLE can be replaced
```suggestion
            <StyledText 
            oddStyle="bodyTextSemiBold"
              desktopStyle="bodyDefaultRegular"
            >{message}</StyledText>
```